x-pack/plugins/security/server/authentication/api_keys.ts

```diff
@@ -164,4 +164,19 @@ export class APIKeys {
 
     return result;
   }
+
+  async areEnabled(): Promise<boolean> {
```

Comment:
__nit:__ I believe `: Promise<boolean>` isn't necessary and should be automatically inferred